docker-compose.yml

```diff
@@ -78,7 +47,7 @@ services:
       - 8081:8081
       # HTTP Gateway
       - 8080:8080
-      # Daemon API -- DO NOT EXPOSE PUBLICLY
+      # Private Daemon API
```

Comment:
Important to specify to not port publicly, not super obvious what 127.0.0.1: directive is doing.

Reply:
I though the `docker-compose.yml` file was only for development purposes anyway?